stl/inc/atomic

```diff
@@ -354,8 +354,14 @@ struct _Atomic_storage {
         _Check_memory_order(_Order);
         const auto _Storage_ptr  = _STD addressof(_Storage);
         const auto _Expected_ptr = _STD addressof(_Expected);
+#if _HAS_CXX20 && !defined(__clang__) /* TRANSITION, LLVM-46685 */ && !defined(__EDG__)

```

Comment:
This occurs repeatedly and is very verbose - should we centralize it to something like `_HAS_BUILTIN_ZERO_NON_VALUE_BITS` defined to be `1` or `0` and then `#undef` at the end?
https://github.com/microsoft/STL/blob/8fcc25fe97bee8fa64232aed5848df0d7c0b6d42/stl/inc/xstring#L339-L343
https://github.com/microsoft/STL/blob/8fcc25fe97bee8fa64232aed5848df0d7c0b6d42/stl/inc/xstring#L460